content/desktop/hardened-desktop/enhanced-container-isolation/limitations.md

```diff
@@ -102,6 +109,13 @@ Desktop.
 are not yet protected by ECI. We expect to improve on this in future versions of
 Docker Desktop.
 
+### Native Windows containers are not supported
+
+ECI only works when Docker Desktop is in Linux containers mode (the default,
+most common mode). It's not supported when Docker Desktop is configured in
+native Windows containers mode (i.e., it's not supported on Windows hosts, when
+the engine is switched from it's default Linux mode to native Windows mode).
```

Comment:
`s/it's/its/` here

Reply:
Maybe `the engine` should be `Docker Desktop` here as well (as the engine itself can't switch Windows <--> Linux)

Reply:
Done, thanks!